src/examples/thlob_example.py

```diff
@@ -18,15 +18,27 @@ def calculate(self):
                 self.bottom.hole(pos=[x, y], diameter=3.1, depth=12, inner=False)
                 self.top.hole(pos=[x, y], diameter=3.1, depth=12, inner=False)
 
-        self.front.box(pos=[self.x_size / 3, 5], width=(self.x_size) / 3, depth=self.y_size / 3 + 2, length=self.z_size - 10, sink=-1)
-        self.left.box(pos=[self.x_size / 3, 5], width=(self.x_size) / 3, depth=self.y_size / 3 + 2, length=self.z_size - 10, sink=-1)
+        self.front.box(
```

Comment:
These changes were caused by formatting the code. Please run `make format` before you push.
I suggest make your changes commit. Then run `make format` look at the automatic code changes and then fix if needed and commit. Then push.